vp/graph.json

```diff
@@ -0,0 +1 @@
+{"directed": true, "multigraph": false, "graph": {}, "nodes": [{"id": 1}, {"id": 2}, {"id": 3}], "links": [{"source": 1, "target": 2}, {"source": 2, "target": 3}]}
```

Comment:
Is this checked in for testing purposes, or by accident?

Reply:
Was an accident. I like your suggestion below re: the file API